server/src/main/java/org/apache/iotdb/db/engine/cache/TsFileMetadataUtils.java

```diff
@@ -98,4 +98,20 @@ public static TsDeviceMetadata getTsDeviceMetaData(String filePath, Path seriesP
     return pathToChunkMetaDataList;
   }
 
+  public static List<ChunkMetaData> getChunkMetaDataList(String sensor,
```

Comment:
Better to use "measurement".

Reply:
Measurement represents a time series, and what I want to express here is the last layer of the time series path.